frontend/src/Utilities/getUniqueElementId.ts

```diff
@@ -1,7 +1,9 @@
 let i = 0;
 
-// returns a HTML 4.0 compliant element IDs (http://stackoverflow.com/a/79022)
-
+/**
```

Comment:
Added this warning since `ClipboardButton` was using it

Could be moved to another PR